updater/registry.go

```diff
@@ -28,12 +28,16 @@ type ResourceRegistry struct {
 	MandatoryUpdates []string
 	AutoUnpack       []string
 
-	Beta    bool
-	DevMode bool
-	Online  bool
+	// UsePreReleases signifies that pre-releases should be used when selecting a
+	// version. This does not affect current releases that are also pre-releases.
+	UsePreReleases bool
```

Comment:
`This does not affect current releases that are also pre-releases.` This one is a bit confusing :D